modules/nf-core/software/fastqc.nf

```diff
@@ -12,6 +12,8 @@ process FASTQC {
 
     container "quay.io/biocontainers/fastqc:0.11.9--0"
 
+    conda (params.conda ? "bioconda::fastqc=0.11.9" : null)
```

Comment:
```
 publishDir "${params.outdir}",
        mode: params.publish_dir_mode,
        saveAs: { filename -> saveFiles(filename:filename, options:options, publish_dir:getSoftwareName(task.process), publish_id:meta.id) }
```
This is the publish dir in the FastQC module from rnaseq dsl2 branch from Harshil. Maybe we can uniform with them

Reply:
That's a good point, I'll have a look into that